e2e-tests/cypress/integration/appellant-submission-supporting-documents.feature

```diff
@@ -0,0 +1,76 @@
+@wip
+Feature: Appellant submission - Supporting documents
+
+  An appeal must include one or many uploaded supporting documents.
```

Comment:
Is this true? I thought an appeal could have zero supporting docs in this section.

Reply:
You're right 'could' is a better word, failed copy paste :/